manifests/globals.pp

```diff
@@ -6,8 +6,10 @@
   $contrib_package_name = undef,
   $devel_package_name   = undef,
   $java_package_name    = undef,
+  $docs_package_name    = undef,
   $perl_package_name    = undef,
   $plperl_package_name  = undef,
+  $plpython_pkg_name    = undef,
```

Comment:
This should be `$plpython_package_name` for consistency with the other parameters. In order to maintain `=` alignment you need to add spaces to all of the other parameters. This makes the change bigger but is necessary for consistency.